modules/layouts/LayoutCS.chpl

```diff
@@ -344,15 +346,17 @@ class CSDom: BaseSparseDomImpl {
   proc dsiLast {
     if _nnz == 0 then return (parentDom.low) - (1,1);
 
-    var _last = parentDom.low[1] - 1;
-    for i in startIdxDom do
-      if startIdx[i] > _last then
-        _last = i-1;
+    const _last_idx = _nnz-1;
```

Comment:
just noticed a few more non-camelCase vars. I'd expect `const _lastIdx`. However, not a big deal.

Reply:
Fixed.  Do the variables starting with underscore represent something?

Reply:
Sometimes leading underscores are used for private methods or fields, but I don't know why they are being used here.